scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/TaskDBUtils.java

```diff
@@ -184,10 +189,23 @@ private static StringBuilder getQueryFilteringExpression(boolean hasUser, boolea
 
         result.append("and taskStatus in (:taskStatus) ");
 
+        if (sortParams != null && !sortParams.getSortParameters().isEmpty()) {
+            result.append("order by ");
+            List<SortSpecifierContainer.SortSpecifierItem> items = sortParams.getSortParameters();
+            for (int i = 0; i < items.size(); i++) {
+                SortSpecifierContainer.SortSpecifierItem item = items.get(i);
+                String order = "ascending".compareTo(item.getOrder().toString()) == 0 ? "ASC" : "DESC";
+                result.append("T." + item.getField() + " " + order);
+                if (i < items.size() - 1) result.append(",");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00122)